pom.xml

```diff
@@ -28,7 +28,7 @@
   </properties>
 
   <scm>
-    <connection>scm:git:git://github.com/${gitHubRepo}.git</connection>
+    <connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
```

Comment:
Adapting to [the deprecation of the `git://` protocol on GitHub](https://github.blog/2021-09-01-improving-git-protocol-security-github/).